doc/overview/project_capabilities.md

```diff
@@ -12,6 +12,15 @@ known to declare the capability.
 | VisualC | Project may contain or compile C++ source files.
 | CSharp | Project may contain or compile C# source files.
 | VB | Project may contain or compile VB source files.
+| JavaScript | Project may contain JavaScript source files.
+| TypeScript | Project may contain TypeScript source files.
+| JavaScriptProject | The project uses JavaScript project system.
```

Comment:
Can we elaborate on "Javascript project system"? That's not a project capability, really, but we'll have to let that slide for historical reasons. But if you can expound on this to explain it is a project that is behind Windows 8+ Javascript projects (and any other that apply) that will make this more specific and actionable.